From 4168ca76abe568c7df898771eb65a484af264b9e Mon Sep 17 00:00:00 2001
From: Richard Marko <rmarko@redhat.com>
Date: Mon, 3 Feb 2014 14:47:21 +0100
Subject: [LIBREPORT PATCH 25/25] Provide SYSLOG_FACILITY when logging through
 journal

Use SYSLOG_FACILITY=1 (user.notice) as a workaround
for rhbz#1026804. This ensures forwarding to rsyslog
works correctly.

Signed-off-by: Richard Marko <rmarko@redhat.com>

Closes rhbz#1063804

Signed-off-by: Jakub Filak <jfilak@redhat.com>
---
 src/lib/logging.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/lib/logging.c b/src/lib/logging.c
index afa480b..259a634 100644
--- a/src/lib/logging.c
+++ b/src/lib/logging.c
@@ -143,6 +143,7 @@ static void log_handler(int level,
                             "CODE_FILE=%s", file,
                             "CODE_LINE=%d", line,
                             "CODE_FUNC=%s", func,
+                            "SYSLOG_FACILITY=1",
                             NULL);
     }
 }
-- 
1.8.3.1

